### ISSUE TITLE
Feature extraction 

### ISSUE THREAD

**Issue by sharidloaiciga** [swiseman/nn_coref#7]
Hi!

Thanks for the code!

I'm trying to run the command to extract the features using the pre-compiled moarcoref-assembly-1.jar, and I get the message below about invalid options:  

`java -jar -Xmx30g modifiedBCS/moarcoref-assembly-1.jar ++modifiedBCS/base.conf -execDir /home/loaiciga/coref/features/  -numberGenderData /home/loaiciga/coref/conll-2012/gender.data -animacyPath /home/loaiciga/coref/conll-2012/animate.unigrams.txt -inanimacyPath /home/loaiciga/coref/conll-2012/inanimate.unigrams.txt -trainPath /home/loaiciga/coref/conll-2012/flat_train_2012 -devPath /home/loaiciga/coref/conll-2012/flat_dev_2012 -testPath /home/loaiciga/coref/conll-2012/flat_test_2012  -mode SMALLER -conjType NONE -pairwiseFeats FINAL+MOARANAPH+MOARP`

`Invalid enum: 'SMALLER'; valid choices: WRITE_FEATS_SEP` 

And using the WRITE_FEATS_SEP option, doesn’t produce the OPCs.txt files. 

Any suggestions?

Thanks again, 
Sharid

**Comment by sharidloaiciga** [comment]
solved!

**Comment by swiseman** [comment]
Hi,

Sorry for the delay: I wasn't able to find a problem with the jar, but did you have to do something special to fix it? (Is it possible that you had an older version of the jar somehow?)

Also, just regarding OPC files, they should already be in the repo if you need them.

**Comment by sharidloaiciga** [comment]
I don't think so. I cloned it some days ago only. But when I tried using it, I only got the WRITE_FEATS_SEP option which produced slightly different features files  (NONE-FINAL+MOARANAPH+MOARP-anaph | pw) and no OPCs (although yes, they're already in the repo). 

So I recompiled it as suggested (an option I initially wanted to avoid because I needed my server admin to install the sbt). At this point there were other problems. Basically I changed addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6") instead of addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.13.0") in assembly.sbt and commented out all the calls to Arab stuff in the source files due to problems with character encodings. Then it worked just fine. 

Thank you all the same, maybe these modifications are useful for someone out there.
